### PR TITLE
[CORE-581] Show latest events on Stream show page

### PR DESCRIPTION
**Pull request by Aapzu**

### CODE REVIEW

File: grails-app/controllers/com/unifina/controller/data/StreamController.groovy [hunk 1]
```diff
@@ -90,15 +90,13 @@ class StreamController {
...
-	// Can be extended to handle more types
-	def details() {
-		getAuthorizedStream(params.id) { stream, user ->
-			def model = [stream: stream, config: (stream.config ? JSON.parse(stream.config) : [:])]
-			render(template: stream.feed.streamPageTemplate, model: model)
+			[
+				stream: stream,
+				writable: writetable,
+				shareable: shareable,
+				key: springSecurityService.currentUser?.keys?.iterator()?.next(),
```
Comment (hpihkala) [inline grails-app/controllers/com/unifina/controller/data/StreamController.groovy]: Key could be fetched from the api

File: web-app/react-app/actions/stream.js [hunk 2]
```diff
@@ -0,0 +1,356 @@
+// @flow
```
Comment (hpihkala) [inline web-app/react-app/actions/stream.js]: Remove stuff that is unnecessary for this PR

File: web-app/react-app/components/ProfilePage/IntegrationKeyHandler/index.jsx [hunk 3]
```diff
@@ -9,15 +9,20 @@ export default class IntegrationKeyHandler extends Component<{}> {
...
-                <Row>
-                    <IntegrationKeyHandlerSegment
-                        service="ETHEREUM"
-                        name="Ethereum"
-                        inputFields={['privateKey']}
-                        tableFields={['address']}
-                    />
-                </Row>
+            <Panel>
+                <Panel.Heading>
```
Comment (hpihkala) [inline web-app/react-app/components/ProfilePage/IntegrationKeyHandler/index.jsx]: This affects Kare's branch, please give him a heads up (or update react-bootstrap in Kare's branch too)

File: web-app/react-app/components/StreamPage/PreviewView/index.jsx [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+                    <Modal.Body>
+                        <Table className={styles.infoScreenModalTable}>
+                            <tbody>
+                                <tr>
+                                    <th>Stream id</th>
+                                    <td>{this.props.stream && this.props.stream.id}</td>
+                                </tr>
+                                <tr>
+                                    <th>Message Timestamp</th>
+                                    <td>{PreviewView.prettyPrintDate(this.state.infoScreenMessage && this.state.infoScreenMessage.metadata && this.state.infoScreenMessage.metadata.timestamp, tz)}</td>
```
Comment (hpihkala) [inline web-app/react-app/components/StreamPage/PreviewView/index.jsx]: `PrettyPrintedDate` component?

File: web-app/react-app/flowtype/actions/stream-actions.js [hunk 5]
```diff
@@ -0,0 +1,83 @@
+// @flow
```
Comment (hpihkala) [inline web-app/react-app/flowtype/actions/stream-actions.js]: Remove stuff that is unnecessary for this PR

File: web-app/react-app/flowtype/states/stream-state.js [hunk 6]
```diff
@@ -0,0 +1,22 @@
+// @flow
```
Comment (hpihkala) [inline web-app/react-app/flowtype/states/stream-state.js]: Remove stuff that is unnecessary for this PR

File: web-app/react-app/flowtype/stream-types.js [hunk 7]
```diff
@@ -0,0 +1,24 @@
+// @flow
```
Comment (hpihkala) [inline web-app/react-app/flowtype/stream-types.js]: Remove stuff that is unnecessary for this PR

File: web-app/react-app/reducers/stream.js [hunk 8]
```diff
@@ -0,0 +1,224 @@
+// @flow
```
Comment (hpihkala) [inline web-app/react-app/reducers/stream.js]: Remove stuff that is unnecessary for this PR

**Comment by hpihkala** [comment]
@Aapzu some minor suggestions still unfixed but we can leave them for UI rewrite. Ok to land? Please resolve conflicts and let's get this merged!

**Comment by Aapzu** [comment]
Resolved the conflicts but couldn't get this to work (some problems with e&e). Could you try it if it works on your machine?

**Pull request merged**